harness/features/allFeatures.local.test.ts

```diff
@@ -0,0 +1,113 @@
+import { 
+    getConnectionStringForProxy, 
+    forEachSDK, 
+    describeIf, 
+    createUser,
+    TestClient,
+    wait
+} from '../helpers'
+import { Capabilities, SDKCapabilities } from '../types'
+import { v4 as uuidv4 } from 'uuid'
+import { getServerScope } from '../nock'
+import { config, expectedFeaturesVariationOn } from '../mockData/config'
+
+jest.setTimeout(10000)
+
+const scope = getServerScope()
+
+describe('allFeatures Tests - Local', () => {
+    forEachSDK((name) => {
+        let url: string
+        const capabilities: string[] = SDKCapabilities[name]
+        
+        let variationOnUser: string
+        let noVariationUser: string
+        let invalidUser: string
+
+        beforeAll(async () => {
+            url = getConnectionStringForProxy(name)  
+            
+            variationOnUser = (
+                await createUser(url, { user_id: 'user1', customData: { 'should-bucket': true } })
+            ).headers.get('location')
+
+            noVariationUser = (
+                await createUser(url, { user_id: 'user3' })
+            ).headers.get('location')
+
+            invalidUser = (
+                await createUser(url, { name: 'invalid' })
+            ).headers.get('location')
+        })
+
+        describeIf(capabilities.includes(Capabilities.local))(name, () => {
+            describe('uninitialized client', () => {
+                const testClient = new TestClient(name)
+
+                beforeAll(async () => {
+                    await testClient.createClient()
+                    const configRequestUrl = `/${testClient.clientLocation}/config/v1/server/${testClient.sdkKey}.json`
+                    const interceptor = scope
+                        .get(configRequestUrl)
+    
+                    interceptor.delay(2000).reply(404)
+    
+                    await wait(2500)
+                    // await waitForRequest( // TODO rebase when variable local tests merged
+                    //     scope,
+                    //     interceptor,
+                    //     3000,
+                    //     'Config request timed out'
+                    // )
+                })
+
+                it('should return empty object if client is uninitialized',  async () => {
+                    const featuresResponse = await testClient.callAllFeatures(variationOnUser, false)
+                    const features = await featuresResponse.json()
+                    expect(features).toMatchObject({})
+                })
+
+                it.failing('should throw exception if user is invalid',  async () => { // TODO fix in node sdk
```

Comment:
is there a ticket for this somewhere? (the nodejs sdk fix)

Reply:
yep https://taplytics.atlassian.net/browse/DVC-5445